### PR TITLE
add additional contact creation for nurses

### PR DESCRIPTION
**Pull request by ericcf**
[#112388915]

### CODE REVIEW

File: config/locales/es-PE.yml [hunk 1]
```diff
@@ -476,6 +476,17 @@ es-PE:
   activerecord:
     errors:
       <<: *errors
+    field_options:
+      additional_contact:
+        kinds:
+          - En persona
```
Comment (eschlange) [inline config/locales/es-PE.yml]: Not a big deal, but the other two translation files have this and the following "kinds" staring with lower case letters.

Comment (ericcf) [inline config/locales/es-PE.yml]: Yeah I noticed that, but I'm just copying/pasting from the Google doc they made... :information_desk_person:

Comment (eschlange) [inline config/locales/es-PE.yml]: Cool.

**Comment by eschlange** [comment]
Looks good. Solid spec coverage.

**Pull request merged**